PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -489,45 +582,140 @@ if [ "$CustomBrain" = "NONE" ] ; then
       fi
 
       # ACPC align T1w or T2w image to specified MNI Template to create native volume space
-      log_Msg "Aligning ${TXw} image to ${TXwTemplate} to create native volume space"
-      log_Msg "mkdir -p ${TXwFolder}/ACPCAlignment"
-      mkdir -p ${TXwFolder}/ACPCAlignment
-      ${RUN} ${HCPPIPEDIR_PreFS}/ACPCAlignment.sh \
-        --workingdir=${TXwFolder}/ACPCAlignment \
-        --in=${TXwFolder}/${TXwImage} \
-        --ref=${TXwTemplate} \
-        --out=${TXwFolder}/${TXwImage}_acpc \
-        --omat=${TXwFolder}/xfms/acpc.mat \
-        --brainsize=${BrainSize}
+
+
+fi
+
+if ["$Runmode" -lt 3 ]; then
+
+  for TXw in ${Modalities} ; do
+    # set up appropriate input variables
+    if [ $TXw = T1w ] ; then
+	TXwInputImages="${T1wInputImages}"
+	TXwFolder=${T1wFolder}
+	TXwImage=${T1wImage}
+	TXwTemplate=${AtlasSpaceFolder}/T1wTemplate	
+	TXwTemplateBrain=${AtlasSpaceFolder}/T1wTemplateBrain
```

Comment:
This treats "T1wTemplate" as a literal string, not a variable name.  I don't think that was intended.  This occurs a few additional times in this file.
```suggestion
	TXwTemplate="$AtlasSpaceFolder"/"$T1wTemplate"
	TXwTemplateBrain="$AtlasSpaceFolder"/"$T1wTemplateBrain"
```